apps/desktop/src/lib/navigation/BranchListingSidebarEntry.svelte

```diff
@@ -92,15 +96,18 @@
 		if (ownedByUser) {
 			const name = (await gitConfigService.get('user.name')) || unknownName;
 			const email = (await gitConfigService.get('user.email')) || unknownEmail;
-			const srcUrl = await gravatarUrlFromEmail(email);
+			const srcUrl = $user?.picture ? $user?.picture : await gravatarUrlFromEmail(email);
```

Comment:
nit: Can we make this a bit more concise, like so:
```ts
$user?.picture ?? (await gravatarUrlFromEmail(email))
```

Also, I guess here we also need to check whether the `$user` email matches the git config's `user.email`, right?

Reply:
Originally my thoughts were that because it's checking the gitconfig, at this point it's definitely "you", so use your profile image. 
However for consistency it does make sense to check the email 🙂

Reply:
It's a good point.
At least for now, I think it makes sense to compare against the email always so that the gitbutler's account picture appears only when using a git profile that matches, and a different picture if you decide to use another git profile (in the case you use different emails for different projects, for example).

Reply:
100% agree - I have this that config locally 😅 I think my GB email is different to both my normal git emails (work and personal) 🙂